.github/workflows/periodic.yml

```diff
@@ -0,0 +1,98 @@
+# Workflow intended to periodically test containerd integration tests
```

Comment:
How about changing the name of the file to `windows-periodic.yml` or `win-periodic.yml` , since the top-level `name` and `env` are Windows-specific?

Reply:
Yes, will do.